### PR TITLE
Isomorphisms between hSets, isUnivalent SET

### PR DESCRIPTION
**Pull request by marcinjangrzybowski**

### CODE REVIEW

File: Cubical/Categories/Functor/Properties.agda [hunk 1]
```diff
@@ -114,3 +116,25 @@ module _ {F : Functor C D} where
         g = F ⟪ f ⟫
         g⁻¹ : D [ y' , x' ]
         g⁻¹ = F ⟪ f⁻¹ ⟫
+
+
+isSet-Functor : isSet (D .ob) → isSet (Functor C D)
```
Comment (mortberg) [inline Cubical/Categories/Functor/Properties.agda:121]: I would call this `isSetFunctor`

File: Cubical/Foundations/Equiv/Properties.agda [hunk 2]
```diff
@@ -198,3 +198,10 @@ isEquivFromIsContr : {A : Type ℓ} {B : Type ℓ′}
 isEquivFromIsContr f isContrA isContrB =
   subst isEquiv (λ i x → isContr→isProp isContrB (fst B≃A x) (f x) i) (snd B≃A)
   where B≃A = isContr→Equiv isContrA isContrB
+
+isEquiv[f∘equivFunA≃B]→isEquiv[f] : {A : Type ℓ} {B : Type ℓ′} {C : Type ℓ′′}
+                 → (f : B → C) (A≃B : A ≃ B)
+                 → isEquiv (f ∘ equivFun A≃B)
+                 → isEquiv f
+isEquiv[f∘equivFunA≃B]→isEquiv[f] f (g , gIsEquiv) f∘gIsEquiv  =
+  precomposesToId→Equiv f _  (cong fst (invEquiv-is-linv (_ , f∘gIsEquiv))) (snd (compEquiv (invEquiv (_ , f∘gIsEquiv) ) (_ , gIsEquiv)))
```
Comment (mortberg) [inline Cubical/Foundations/Equiv/Properties.agda:207]: Long line

**Review by mortberg — CHANGES_REQUESTED** [review]
Two small changes requested, then I would be happy to merge!

**Comment by marcinjangrzybowski** [comment]
just saw bunch of similar things merged from one of the previous PRs, I will check if this still anything new in this PR

**Comment by mortberg** [comment]
> just saw bunch of similar things merged from one of the previous PRs, I will check if this still anything new in this PR

Ok, I looked as well but didn't find anything. But I could possibly have missed something

**Comment by marcinjangrzybowski** [comment]
Ok, I thought is saw some overlap with `CommRing category is univalent and complete #694`, but I think I was mistaken.

**Comment by mortberg** [comment]
> Ok, I thought is saw some overlap with `CommRing category is univalent and complete #694`, but I think I was mistaken.

Yes, I thought so too because I could not remember what I had done in that PR... But I think it's all fine and there's no overlap

File: Cubical/Foundations/Equiv/Properties.agda (hunk 2 above)
Comment (marcinjangrzybowski) [reply Cubical/Foundations/Equiv/Properties.agda]: done

**Pull request merged**